packages/instant-apps-components/src/instant-apps-app-guide.html

```diff
@@ -0,0 +1,57 @@
+<!-- /*
+*   Copyright (c) 2024 Esri
+*   All rights reserved under the copyright laws of the United States and applicable international laws, treaties, and conventions.
+*   This material is licensed for use under the Esri Master License Agreement (MLA), and is bound by the terms of that agreement.
+*   You may redistribute and use this code without modification, provided you adhere to the terms of the MLA and include this copyright notice.
+*   See use restrictions at http://www.esri.com/legal/pdfs/mla_e204_e300/english
+*/ -->
+
+<!DOCTYPE html>
+<html dir="ltr" lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0" />
+  <title>ArcGIS Instant Apps Components</title>
+  <link rel="stylesheet" href="https://js.arcgis.com/4.30/esri/themes/light/main.css" />
+  <script src="https://js.arcgis.com/4.30/"></script>
+  <script type="module" src="https://js.arcgis.com/map-components/4.30/arcgis-map-components.esm.js"></script>
+  <script type="module" src="https://js.arcgis.com/calcite-components/2.12.2/calcite.esm.js"></script>
+  <link rel="stylesheet" type="text/css" href="https://js.arcgis.com/calcite-components/2.12.2/calcite.css" />
+  <script type="module" src="/build/instant-apps-components.esm.js"></script>
+  <script nomodule src="/build/instant-apps-components.js"></script>
+
+  <style>
+    html,
+    body {
+      box-sizing: border-box;
+      width: 100%;
+      height: 100%;
+      margin: 0;
+      padding: 0;
+    }
+  </style>
+</head>
+
+<body>
+  <arcgis-map item-id="8891ee1d2e0e428bb96c58b8ecf8c408">
+    <arcgis-expand position="top-right">
+      <arcgis-placement>
+        <instant-apps-app-guide></instant-apps-app-guide>
+      </arcgis-placement>
+    </arcgis-expand>
+    <arcgis-legend position="bottom-left" />
+    <arcgis-zoom position="top-left" />
+    <arcgis-home position="top-left"></arcgis-home>
+  </arcgis-map>
+
+  <script>
+    async function load() {
+      const esriConfig = await $arcgis.import("esri/config");
+      esriConfig.portalUrl = "https://nw-brews.mapsdevext.arcgis.com";
```

Comment:
Can you remove the devext url from this sample

Reply:
Done!